flow/project.py

```diff
@@ -1687,7 +1684,7 @@ def run(self, jobs=None, names=None, pretend=False, np=None, timeout=None, num=N
 
         This function will run in an infinite loop until all pending operations
         have been executed or the total number of passes per operation or the total
-        number of exeutions have been reached.
+        number of executions have been reached.
```

Comment:
I have trouble parsing this parallel structure.
I propose:
This function will run in an infinite loop until executing all
pending operations, unless it reaches the maximum number of
passes per operation or the maximum number of executions.

Reply:
Good rewrite. I committed that.